glyphs-reader/src/font.rs

```diff
@@ -1862,6 +1885,31 @@ impl RawFeature {
     fn disabled(&self) -> bool {
         self.disabled == Some(1)
     }
+
+    // https://github.com/googlefonts/glyphsLib/blob/24b4d340e4c82948ba121dcfe563c1450a8e69c9/Lib/glyphsLib/builder/features.py#L134
+    fn feature_names(&self) -> String {
+        if self.labels.is_empty() {
+            String::new()
+        } else {
+            let labels = self
+                .labels
+                .iter()
+                .map(|label| {
+                    let language_id = GLYPHS_TO_OPENTYPE_LANGUAGE_ID
+                        .iter()
+                        .find(|entry| entry.0 == label.language)
+                        .map(|entry| entry.1)
+                        .unwrap_or_else(|| {
+                            panic!("Unknown feature label language: {}", label.language);
```

Comment:
I don't think we should panic here; instead I would skip this label and emit a warning. You can use `filter_map` instead of `map` to skip missing items.